libs/shared/base-editor-components/src/create/create.component.spec.ts

```diff
@@ -107,6 +109,15 @@ describe('CreateComponent', () => {
     expect(selectSpy).toHaveBeenCalledWith(true, id);
   });
 
+  it('does not allow a higher value than max value', () => {
+    const unallowedIdValue = MAX_INT_UNSIGNED_VALUE + 1;
+    component.idModel = unallowedIdValue;
+
+    component['checkMaxValue']();
```

Comment:
let's not do this unless strictly necessary. Isn't is possible to trigger this from the UI?

Reply:
I tried but it was not feasible